src/main/java/com/shevtsov/dao/ClientDao.java

```diff
@@ -3,5 +3,41 @@
 import com.shevtsov.domain.Client;
 
 public interface ClientDao {
+
+    /**
+     * Saves Client-object to DB.
+     * @param client the Client-object, which is saved.
+     * @return true if new client was saved successfully, or false otherwise.
+     */
     boolean saveClient(Client client);
```

Comment:
In case that your interface processed only Client, you can avoid to  put Client in every methods name

Reply:
done